modules/ROOT/pages/_partials/upgrade-tool.adoc

```diff
@@ -0,0 +1,82 @@
+//This content is reused in:
+//mule-upgrade-tool.adoc
+
+// Before You Begin
+// tag::BeforeYouBegin[]
+
+* Ensure that you have xref:release-notes::mule-upgrade-tool/mule-upgrade-tool.adoc[the latest available version of Mule upgrade tool] so that the tool runs with the latest fixes and security enhancements.
+* A currently operational Mule instance running version 4.1.1 or later, in _stopped_ status to prepare for the upgrade.
++
+When updating your current Mule instance from a version between 4.1.1 and 4.1.4, use any patch update released after January 20, 2022. If the running version is not one of the previously mentioned, an error message appears stating that the Mule version cannot be upgraded unless an upgrade is made.
```

Comment:
```suggestion
When updating your current Mule instance from a version between 4.1.1 and 4.1.4, upgrade from any patch update released after January 20, 2022. Earlier releases of versions 4.1.1 through 4.1.4 are not supported, and attempts to upgrade them produce an error message stating that the Mule version cannot be upgraded without first upgrading to a supported version.
```

Reply:
Are we saying something like the above?

Reply:
And do we need to point them to other docs for upgrading those unsupported versions to a supported one?

Reply:
actually versions between 4.1.1 and 4.1.4 are supported for versions released after january 2022, so if you have any of those versions, with those dates on it, you will be able to update. For prior versions you will need to follow the manual procedure that was part of the original document. Perhaps we need to add that link here.

Reply:
@dmerlob I added a link in the commit suggestion above.